helpers/apolloClient.ts

```diff
@@ -18,10 +18,7 @@ const whiteList = ['/curriculum']
 
 //there is no global await and this logic can't be refactored into other functions because pages/_app.tsx can't be async
 ;(async function () {
-  if (
-    typeof window !== 'undefined' &&
-    whiteList.includes(window.location.pathname)
-  ) {
+  if (whiteList.includes(window.location.pathname)) {
```

Comment:
I think this is why Vercel build throw errors. The code first runs server-side and at that period, `window` is undefined.

The past check of whether `window` is undefined or not has to return.

```suggestion
 if (
    typeof window !== 'undefined' &&
    whiteList.includes(window.location.pathname)
    )
```

Reply:
Thanks!